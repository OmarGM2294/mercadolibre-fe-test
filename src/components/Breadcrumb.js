import styles from '../../styles/components/Breadcrumb.module.scss'

const Breadcrumb = (props) => (
  <div className={styles.breadcrumb}>
    { props.data.map((category, index) => (
      <div className={styles.container} key={index}>
        <span>
          { category }
        </span>
        <>      
          { props.data.length -1 !== index ?
            <span className={styles.separator}>
              { '>' }
            </span>
          : null }
        </>
      </div>
    )) }
  </div>
)

export default Breadcrumb