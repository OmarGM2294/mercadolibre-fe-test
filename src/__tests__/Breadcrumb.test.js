import React from 'react'

import { create } from 'react-test-renderer'

import Breadcrumb from '../components/Breadcrumb'

describe('Breadcrumb', () => {
  it('Should match snapshot (it renders correctly)', () => {
    const component = create(<Breadcrumb data={[]}/>)

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})