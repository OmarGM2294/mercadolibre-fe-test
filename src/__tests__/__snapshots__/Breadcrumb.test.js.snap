// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb Should match snapshot (it renders correctly) 1`] = `<div />`;
